src/controllers/courseController.ts

```diff
@@ -59,6 +59,9 @@ export class CourseController {
 		this.router.get('/content-management/courses/details/:courseId?', this.getCourseDetails())
 		this.router.post('/content-management/courses/details/:courseId?', this.setCourseDetails())
 
+		this.router.get('/content-management/courses/:courseId/sort-modules?', this.sortModules())
+
+		this.router.post('/content-management/courses/:courseId/status', this.setStatus())
 		this.router.get('/content-management/courses/:courseId/sortDateRanges-modules?', this.sortModules())
```

Comment:
I know this line hasn't been changed in that PR - but still, same comment as above